zulip/integrations/google/doc.md

```diff
@@ -1,72 +1,172 @@
-    to get a `client_secret` file. Save this file as `client_secret.json`
-    to your `~/` directory.
+### Configure the integration
 
-1.  Next, install the latest Google API Client for Python by following the
-    instructions on the
-    [Google website](https://developers.google.com/api-client-library/python/start/installation).
+The integration can be configured by:
 
-1.  In Zulip, go to your click on the cog in the top right corner, and
-    then clicking on **Personal settings**.
+* Passing command line arguments to the integration script.
+* Editing the `google-calendar` section of the `zuliprc` file.
 
-1.  Click on the tab that’s labeled **Account & privacy** and click on
-    **Manage your API key**. Enter your password if prompted, and
-    download the `zuliprc` file. Save this file as `.zuliprc` to your `~/`
-    directory.
+The configuration settings in `zuliprc` will be overridden by the
+corresponding command line options, if both are used.
 
-    ![Download zuliprc file](/static/images/integrations/google/calendar/001.png)
+### Configuration options
 
-1.  Run the `get-google-credentials` with this command:
+The integration script accepts the following configuration options:
 
-        python /usr/local/share/zulip/integrations/google/get-google-credentials
+* `calendar`: The `calendar ID` of the Google calendar to get events from.
+By default, the `primary` calendar is used.
 
-1.  It should open up a browser and ask you for certain permissions. Give
-    Zulip access, and move on to the next step. If it doesn’t open a
-    browser, follow the instructions in the terminal window.
+* `interval`: How many minutes in advance you want reminders delivered.
+The default value is 30 minutes.
 
-1.  Now, all that’s left to do is to run the `gcal-bot` script, in the
-    same directory as the `get-google-credentials` script, with the
-    necessary parameters:
+* `channel`: The name of the Zulip channel you want to receive
+notifications in. By default, messages are sent to the DMs of the bot
+owner.
 
-        python /usr/local/share/zulip/integrations/google/gcal-bot --user foo@zulip.com
+* `topic`: The name of the Zulip topic you want to receive notifications
+in. This option is ignored if the `channel` option is unspecified. If the
+`channel` option is specified but the `topic` option is not, notifications
+are sent to a topic named "calendar-reminders" in the channel.
 
-    The `--user` flag specifies the user to send the reminder to.
+* `client-secret-file`: The path to the file containing the client secret.
+By default, the client secret file is assumed to be at
+`~/client_secret.json`.
 
-1.  Don’t close the terminal window with the bot running (you can use
-    `screen` if needed). You will only get reminders if the bot is still
-    running.
+* `tokens-file`: The path to the file where the OAuth tokens are stored. By
+default, the tokens file is generated at `~/google-tokens.json` after the
+first run.
```

Comment:
I assume this is the first run of the integration script, correct? It might be clearer to say something like "when the integration script is first run".

When would it make sense to specify this option? Does running the script ever update the tokens?